ExtractorUtils/RawExtractorManager/RawExtractorManager.cs

```diff
@@ -0,0 +1,237 @@
+using System;
+using System.Threading;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Collections.Generic;
+using CogniteSdk;
+using Cognite.Extractor.Common;
+using Microsoft.Extensions.Logging;
+
+namespace Cognite.Extractor.Utils
+{
+    /// <summary>
+    /// Class used to manage an extractor using a Raw database.
+    /// </summary>
+    public class RawExtractorManager : IExtractorManager
```

Comment:
Perhaps add `HA` to the name, i.e. `RawHighAvailabilityManager`, and `IHighAvailabilityManager`. This is extractor utils, after all, the `Extractor` bit is fairly implicit.